src/app/services/environments.service.ts

```diff
@@ -377,6 +377,36 @@ export class EnvironmentsService {
     }
   }
 
+  /**
+   * Start / stop all environments
+   */
+  public toggleAllEnvironments() {
+    const environments = this.store.get('environments');
+    const environmentsStatus = this.store.get('environmentsStatus');
+
+    // check if environments should be started or stopped. If at least one env is turned off, we'll turn all on
+    const shouldStart = Object.keys(environmentsStatus)
+      .map(uuid => environmentsStatus[uuid].running)
+      .some(status => !status);
+
+    environments.map(environment => {
+      const environmentState = environmentsStatus[environment.uuid];
+      this.logger.info(
+        `start toggle environment ${environment.name}`,
+      );
```

Comment:
You can remove this log. It's redundant with the "starting server" logged in the server.service